packages/office-addin-cli/src/npmPackage.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import readPackageJson, { ScriptsObject } from "read-package-json-fast";
+
+let scripts: ScriptsObject | undefined;
+
+export function clearCachedScripts() {
+    scripts = undefined;
+}
+
+async function readPackageJsonScripts(jsonPath: string) {
+    const packageJson = await readPackageJson(`${jsonPath}`);
+    return packageJson.scripts || {};
```

Comment:
If package.json doesn't contain "scripts", it will ensure an empty object is returned.